convert2rhel/actions/system_checks/is_loaded_kernel_latest.py

```diff
@@ -80,6 +85,15 @@ def run(self):  # pylint: disable= too-many-return-statements
                 "the %s comparison.\n"
                 "Beware, this could leave your system in a broken state." % package_to_check
             )
+            self.add_message(
+                level="WARNING",
+                id="UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK_DETECTED",
+                message=(
+                    "Detected 'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK' environment variable, we will skip "
+                    "the %s comparison.\n"
+                    "Beware, this could leave your system in a broken state." % package_to_check
+                ),
+            )
```

Comment:
Remember to remove the logger.warning above
```suggestion
            message = (
                    "Detected 'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK' environment variable, we will skip "
                    "the %s comparison.\n"
                    "Beware, this could leave your system in a broken state." % package_to_check
                )
            logger.warning(message)
            self.add_message(
                level="WARNING",
                id="UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK_DETECTED",
                message=message,
            )
```